src/themes/custom_theme/custom_theme.js

```diff
@@ -0,0 +1,246 @@
+			// For creating array.
+			if(property === "arr"){
+				temp = temp + "[" + "\"rgba(" + rgba.join(", ") + ")\"" + "]";
+				return;
+			}
+			end = end + "}";
+			if(index === arr.length - 1){
+				temp = temp + "{" + "\"" + property + "\"" + ":" + "\"rgba(" + rgba.join(", ") + ")\"";
+				return;
+			}
+			temp = temp + "{" + "\"" + property + "\"" + ":";
+		});
+		temp = temp + end;
+		$.extend(themeConf, JSON.parse(temp));
+	}
+
+	$('a.theme_custom')
+    .off('click')
+    .on('click', function(){
+
+    	var ele = $(this);
+
+    	require(['css!themes/custom_theme/custom_theme.css']);
+
+      require(['text!themes/custom_theme/custom_theme.html'], function($html){
+      	$html = $($html);
+
+      	$html.find(".color_input_width").each(function(index, value){
+      		var id = $(value).attr("id").replace("theme_",""),
+      				alpha = $(value).attr("alpha");
+      		$(value).colorpicker({
+      			part:	{
+                    map:		{ size: 128 },
+                    bar:		{ size: 128 }
+                },
+                select:	function(event, color) {
+                    $(value).css({
+                        background: '#' + color.formatted
+                    }).val('');
+                    createThemeObject(id, color.formatted, alpha);
+										updateChart();
+                },
+                ok: function(event, color) {
+                    $(value).css({
+                        background: '#' + color.formatted
+                    }).val('');
+                    createThemeObject(id, color.formatted, alpha);
+                    updateChart();
+                }
+      		});
+      	});
+
+      	win = windows.createBlankWindow($html,
+      		{
+      			autoOpen: false,
+            resizable: false,
+            collapsable: false,
+            minimizable: false,
+            maximizable: false,
+            closable: false,
+            closeOnEscape: false,
+            width: 650,
+            height: 515,
+            title: "Customize chart appearance",
+            modal: true,
+            my: 'center',
+            at: 'center',
+            of: window,
+            destroy: function() {
+                win = null;
+            },
+            buttons: {
+              Apply: function() {
+              		openConfirmDialog();
+              },
+              Cancel: function() {
+                  $( this ).dialog( 'close' );
+                  $( this ).dialog( "destroy" );
+                  var themeName = local_storage.get("theme");
+                  themeName = themeName && themeName.name;
+                  //Reseting charts to previous theme or default.
+							    if (themeName) {
+							        require(['lib/highstock/themes/' + themeName]);
+							    } else{
+							    	Highcharts.setOptions({});
+							    }
+              }
+            }
+      		});
+
+    		win.dialog('open');
+    		generateData();
+    		createChart();
+      });
+  });
+
+  function generateData() {
```

Comment:
@4p00rv Instead of random numbers, please hard code some nice OHLC code. You can get it easily from our developers.binary.com. Just get 10-20 values. We need candles and not line